src/include/mpir_attr.h

```diff
@@ -202,4 +202,6 @@ void MPID_Attr_free(MPIR_Attribute * attr_ptr);
 int MPIR_Call_attr_delete(int, MPIR_Attribute *);
 int MPIR_Call_attr_copy(int, MPIR_Attribute *, void **, int *);
 
+void MPIR_free_keyval(MPII_Keyval * keyval_ptr);
```

Comment:
It was, but then I squashed them at some point :). Each chapter have some refactoring along with script generation and those misc refactoring itself often cover separate changes may benefit from splitting. That will make this PR way too complex.

Alternatively, I can split this PR into several pieces -- like I did with pt2pt chapter earlier. Is that more preferable? -- Let me know and I'll do that.

The concern is that individual PR takes similar amount of time to review and merge so splitting into 10 or PRs may get the process really long. I'll need some active engagement from reviewers to make the whole thing merged in a reasonable time frame.

BTW, @shintaro-iwasaki I was only expecting you to review the `info` chapter. Appreciate that you are looking at all the commits!

Reply:
I am okay to split this PR into several small PRs, but I don't know what the other reviewers think since we might want to finish this review asap; this PR is blocking the other MPI-4 related PRs that affect the interface.

Anyway I will check only `info` in addition to this `attr` that I already reviewed, but in my understanding every chapter needs to be checked as far as I look through the `attr` chapter.

Reply:
fixed in #4953